Readme.txt

```diff
@@ -195,108 +170,110 @@ The log window marks incorrect entries in your log as follows:
   DUP - duplicate QSO.
 
   NIL - not in other station's log: you made a mistake in the callsign, or forgot
-    to send the corrected call to the station.
+        to send the corrected call to the station.
 
   RST - incorrect RST in your log.
 
   NR - incorrect exchange number in your log.
 
+  CL - incorrect Arrl Field Day Classification in your log.
 
+  NAME - incorrect Name in your log.
 
+  SEC - incorrect ARRL Section in your log.
 
+  ST - incorrect State in your log.
 
 SUBMITTING YOUR SCORE
 
 If you complete a full 60-minute session in the WPX Competition mode, Morse Runner
-will generate a score string that you can post to the Score Board on the web:
-<http://www.dxatlas.com/MorseRunner/MrScore.asp>. Copy and paste your score
-string into the box on the web page and click on the Submit button.
-
-You can view your previous score strings using the File -> View Score menu
-command.
-
-
-
-
-
+will generate a score string that you can post. 
+
+  The original scoring website by Alex Shovkoplyas, VE3NEA, was 
+  http://www.dxatlas.com/MorseRunner/MrScore.asp
+  but it is no longer active. 
+  
+  Lin Quan, BG4FQD, created the scoring website, https://www.bh1scw.com/mr/score
+  but to submit scores you must now email him at bh1scw[at]gmail.com
+  
+  "Open 2019 UZ2M Morse Runner contest" Facebook Group
+  (Still active)
+      Open contest between friends in Morse Runner in two modes, 
+      Single Call and Pile-Up mode. Try 10 min training and fix 
+      screenshot with date and time. Please send a screenshot to 
+      this group and after we check it we will publish it!
+  
+  "ZS-CW Morse Runner" Facebook Group
+      In Morse Runner, Here you will screen shot your 10 min stint, 
+      (ONLY 10 min not more) paste it into this Facebook Group, 
+      the idea is to try beat the previous score of the last person 
+      who posted their screen shot. I'm aware we are all on different 
+      levels of copy speed, but that's irrelevant because you might 
+      want to match their speed, and try beat their score
+
+  "CW Freak - Morse Runner" Facebook Group
+      This is a group for Morse Games fans. I thank in advance anyone 
+      who wants to share their results, their advice and their 
+      impressions with fun.
+
+You can view your previous score strings using the 
+File -> View Score menu command.
 
 VERSION HISTORY
 
-1.80 (W7SST)
+1.80 (W7SST) Oct 2022
```

Comment:
Given our new focus on a Community Edition with multiple authors, I think we should recognize the author of each contribution, not the release. For example, N1MM's release notes recognizes individual contributions by adding the developer's callsign to each enhancement. It looks like they even recognize the callsign who reported the issue. Here is an example from one of their recent release notes:

>Version 1.0.9663 (September 27, 2022)
> 1. WA QSO Party: Rules contest time change. (N7EPD) (Coded by K3CT)
> 2. ARRL 10 GHz: Calls contained within other calls were not being counted as unique calls. E.g W3IP & W3IPA (VE3KH) (Coded by N1MM)
> 3. CQWWRTTY: Change band change counter for multi-1 & 2 to 8 (K9CT) (Coded by N1MM)

We should adopt something similar with the callsign on each contribution, not the release. How does this look?
```
Version 1.80 (Oct 2022)
  - Beta release of multi-contest support (Coded by W7SST)
  - Added ARRL Field Day contest (Coded by W7SST)
  - Added NCL NAQP contest (Coded by W7SST)
```
I recommend that we adopt this new convention starting at v1.80. The prior versions can stay as you have them since each release was created by individual authors.

Reply:
I like this too!

Reply:
I updated Readme.txt to reflect this change.